src/olympia/abuse/cinder.py

```diff
@@ -515,10 +515,10 @@ def flag_for_human_review(self, *, related_versions, appeal=False, forwarded=Fal
         from olympia.reviewers.models import NeedsHumanReview
 
         waffle_switch_name = (
-            'dsa-appeals-review'
-            if appeal
-            else 'dsa-cinder-forwarded-review'
+            'dsa-cinder-forwarded-review'
             if forwarded
+            else 'dsa-appeals-review'
+            if appeal
```

Comment:
mmm, was this change necessary?

Reply:
If `CINDER_APPEAL_ESCALATION` is meant to behave virtually identical to `CINDER_ESCALATION` (beyond the history), it would make sense for this waffle switch name to be the same, no? If it's the other way around, `CINDER_APPEAL_ESCALATION` would have 'dsa-appeals-review' as the name instead.

Reply:
I can see the logic (and don't disagree), but one of reasons we have these series of waffles switches it the rollout is being phased, and reviewers handling appeals is a later phase than handling forwarded reports.  So for now I we're going to have to stick with the inconsistency.